<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Advanced</class>
 <widget class="QMainWindow" name="Advanced">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hLayoutTitle">
     <item>
      <widget class="QLabel" name="ReprojectionInformation">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Reprojection Information</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="editInputParams">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Edit Input Parameters</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hLayoutStatus">
     <item>
      <widget class="QLabel" name="labelStatus">
       <property name="text">
        <string>Status/Error Info</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>621</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabRasterInfo">
     <attribute name="title">
      <string>Raster Info</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>591</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="hLayoutFile">
       <item>
        <widget class="QLabel" name="labelFileName">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>File Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="fileName">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>veg_geographic_1deg.img</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>591</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="hLayoutRowCol">
       <item>
        <widget class="QLabel" name="labelRows">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Rows</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Rows">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>15</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelCols">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Columns</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Cols">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>591</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="hLayoutPixels">
       <item>
        <widget class="QLabel" name="labelPixelSize">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Pixel Size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pixelSize">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="pixelUnits">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Arcsecond(s)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Arcminute(s)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Degree(s)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Meter(s)</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>15</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelPixelType">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Pixel Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="pixelType">
         <property name="minimumSize">
          <size>
           <width>165</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>165</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>7</number>
         </property>
         <item>
          <property name="text">
           <string>Signed 8-Bit Integer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Unsigned 8-Bit Integer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Signed 16-Bit Integer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Unsigned 16-Bit Integer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Signed 32-Bit Integer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Unsigned 32-Bit Integer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Signed 32-Bit IEEE Float</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Signed 64-Bit IEEE Float</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>591</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="hLayoutFill">
       <item>
        <widget class="QLabel" name="labelFillValue">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Fill Value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fillEnable">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="FillValue">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>591</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="hLayoutNoData">
       <item>
        <widget class="QLabel" name="labelNoDataValue">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>No Data Value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="noDataValueEnable">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="noDataValue">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>591</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="hLayoutSheroid">
       <item>
        <widget class="QLabel" name="labelSheroid">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Sheroid</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="sheroidSelector">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Sphere of Radius 6,370,997 meters</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>591</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="hLayoutUL">
       <item>
        <widget class="QLabel" name="labelULlat">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>UL Latitude</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Latitude">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>15</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelULlon">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>UL Longitude</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Longitude">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabProjectionInfo">
     <attribute name="title">
      <string>Projection Info</string>
     </attribute>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>591</width>
        <height>321</height>
       </rect>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_Reprojection"/>
    <addaction name="separator"/>
    <addaction name="actionSelection_Screen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionLoad_Projection_Info"/>
    <addaction name="actionSave_Projection_Info"/>
    <addaction name="separator"/>
    <addaction name="actionToggle_Preview"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionEdit_Author"/>
    <addaction name="actionUser_Guide"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_dRasterBlaster"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/document-open-516.png</normaloff>:/Images/document-open-516.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSave_Reprojection">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/document-save-516.png</normaloff>:/Images/document-save-516.png</iconset>
   </property>
   <property name="text">
    <string>Save Reprojection</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSelection_Screen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/mail-mark-task.png</normaloff>:/Images/mail-mark-task.png</iconset>
   </property>
   <property name="text">
    <string>Selection Screen</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/document-close-316.png</normaloff>:/Images/document-close-316.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionLoad_Projection_Info">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/svn-commit.png</normaloff>:/Images/svn-commit.png</iconset>
   </property>
   <property name="text">
    <string>Load Projection Info</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSave_Projection_Info">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/svn-update.png</normaloff>:/Images/svn-update.png</iconset>
   </property>
   <property name="text">
    <string>Save Projection Info</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionToggle_Preview">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/system-search-4.png</normaloff>:/Images/system-search-4.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Preview</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionEdit_Author">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/configure-2.png</normaloff>:/Images/configure-2.png</iconset>
   </property>
   <property name="text">
    <string>Edit Author Info</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionUser_Guide">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/documentation.png</normaloff>:/Images/documentation.png</iconset>
   </property>
   <property name="text">
    <string>User Guide</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionAbout_dRasterBlaster">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/icon_16x16x32.png</normaloff>:/Images/icon_16x16x32.png</iconset>
   </property>
   <property name="text">
    <string>About dRasterBlaster</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/qt-logo.png</normaloff>:/Images/qt-logo.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
